docs/user/exports.rst

```diff
@@ -44,30 +44,25 @@ Archive course/exam
 -------------------
 Export all course/exam data, including all exercises and student submissions.
 To archive a course or an exam the end date of the entity needs to be in the past.
-You can archive a course by clicking |archive_course| on the course management overview page or an exam by clicking |archive_exam| on the exam checklist page. This will create a zip file containing all relevant data.
+You can archive a course by clicking |archive_course| on the course management overview page or an exam by clicking |archive_exam| on the exam checklist page. This will create a zip file containing all exercises of the exam or course and all student submissions. For a course all exams are exported as well. For each exercise the problem statement, a JSON file with exercise details such as points and the problem statement are exported.
```

Comment:
```suggestion
You can archive a course by clicking |archive_course| on the course management overview page or an exam by clicking |archive_exam| on the exam checklist page. This will create a zip file containing all exercises of the exam or course and all student submissions. For a course all exams are exported as well. For each exercise the a JSON file with exercise details such as points and the problem statement are exported.
```
2x problem statement